builder/builder.go

```diff
@@ -852,3 +875,66 @@ func parseUint32(s string) (uint32, error) {
 	}
 	return uint32(parsed), nil
 }
+
+func createCompressedArchive(dst string, srcs ...string) (err error) {
```

Comment:
Should this be exported so we can use it elsewhere in the code?

Reply:
This function wasn't written with the intent of being exported. If we see value later in exporting it, we can revisit it to make it so.